packager/app/manifest_flags.cc

```diff
@@ -37,3 +37,8 @@ ABSL_FLAG(std::string,
           "",
           "Same as above, but this applies to text tracks only, and "
           "overrides the default language for text tracks.");
+ABSL_FLAG(bool,
+          force_cl_index,
+          false,
```

Comment:
Let's default this to true.  We are about to release v3.0.0, where it is okay to introduce a change that is not strictly backward compatible.  I think some specific order is better than no specific order.

Reply:
I edited this to set the default to true for the CLI.  Let me know if you disagree.